### ISSUE TITLE
Fullscreen mode or window resize

### ISSUE THREAD

**Issue by sebhaase** [leaningtech/cheerpj-meta#78]
I have a Java app where the window (I assume its using Swing) gets cut off at the bottom.  When I increase the parameters given in `cheerpjCreateDisplay(1280,760)` I get more white space around the Java window inside the CheerpJ frame, but it stays cut off at the same size as before. Is the a way to tell the window to go fullsize or to have it get a resize event/trigger.

**Comment by alexp-sssup** [comment]
You should be able to resize the window manually, and the window should also properly resize if you programmatically resize it from the Java side. Does any of this solutions work?

**Comment by sebhaase** [comment]
Manual resizing does in fact help (Thanks!).
I do _not_ have access to the Java source code - I have only the jar-file. So I was hoping the CherpJ [runtime API](https://github.com/leaningtech/cheerpj-meta/wiki/Runtime-API) would allow me to programmatically resize the window from the **_Javascript_** side.  Is that possible? (Manual resizing would be quite difficult to explain to the users!)

**Comment by alexp-sssup** [comment]
No, at this time there is no API to achieve that. Eventually we plan to add a standard "Maximixe" control to the window bar, but we don't have a timeline for this feature right now. Please understand that we are a small company and we need to prioritize feature requests coming from our commercial users.

**Comment by alexp-sssup** [comment]
The maximize button is now implemented and it will be available in the upcoming CheerpJ 3.1 release
